src/pages/ReimbursementAccount/BeneficialOwnersStep.js

```diff
@@ -51,35 +55,55 @@ class BeneficialOwnersStep extends React.Component {
             certifyTrueInformation: getDefaultStateForField(props, 'certifyTrueInformation', false),
             beneficialOwners: getDefaultStateForField(props, 'beneficialOwners', []),
         };
+
+        // These fields need to be filled out in order to submit the form (doesn't include IdentityForm fields)
+        this.requiredFields = [
+            'acceptTermsAndConditions',
+            'certifyTrueInformation',
+        ];
+
+        // Map a field to the key of the error's translation
+        this.errorTranslationKeys = {
+            acceptTermsAndConditions: 'beneficialOwnersStep.error.termsAndConditions',
+            certifyTrueInformation: 'beneficialOwnersStep.error.certify',
+        };
+
+        this.clearError = inputKey => clearError(this.props, inputKey);
+        this.getErrorText = inputKey => getErrorText(this.props, this.errorTranslationKeys, inputKey);
+    }
+
+    /**
+     * @returns {Object}
+     */
+    getErrors() {
+        return lodashGet(this.props, ['reimbursementAccount', 'errors'], {});
+    }
+
+    /**
+     * @param {Number} ownerIndex
+     * @returns {Object}
+     */
+    getBeneficialOwnerErrors(ownerIndex) {
```

Comment:
NAB, with the new changes I'm not sure if this function is really valuable anymore since we only use it once and probably could replace the usage with calling lodashGet directly.

Reply:
Removed